doc/pages/Manual.md

```diff
@@ -52,7 +52,9 @@ following hierarchy:
   they override what may already be present in the `opam` file
 - [`/packages/<pkgname>/<pkgname>.<version>/files/`](#files): contains files
   that are copied over the root of the source tree of the given package before
-  it gets used.
+  it gets used. Before opam 2.2 every files in this directory were copied but
+  since opam 2.3 only the files listed in the
+  [`extra-files`](#opamfield-extra-files) field are copied.
```

Comment:
```suggestion
  it gets used. Before opam 2.3, all files in this directory were copied. 
  Since opam 2.3 _only_ the files listed in the
  [`extra-files`](#opamfield-extra-files) field are copied.
```